recipes/neddy/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "neddy" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 414ca082adde63a513eaea7ae5f789bf7903abe6aa13a428092748b00c212041
+
+build:
+  number: 0
+  entry_points:
+    - neddy=neddy.cl_utils:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.9
+  run:
+    - fundamentals
+    - python >=3.9
+    - pyyaml
+    - astrocalc
```

Comment:
It looks like `astrocalc` isn't available on conda-forge. If you want to add it in this PR, that would solve the issue. See the following for some info on submitting a PR with two packages:
https://conda-forge.org/docs/maintainer/adding_pkgs.html#adding-multiple-packages-at-once

Or you can open another PR for `astrocalc`.